### ISSUE TITLE
Not working with swift 2.0

### ISSUE THREAD

**Issue by mostafamraafat** [limhowe/LimPicker#2]
After cloning it, It had many compilation issues with Swift 2.0
Tried solving it. but they are alot
Your support is highly appreciated
Thank you

**Comment by thihaaung6245** [comment]
Yeah me too...Please help! Also pod support is really appreciated.And Instructions and customization for this library too.
